tests/functional/Exasol/ExasolBaseTestCase.php

```diff
@@ -33,7 +34,7 @@ class ExasolBaseTestCase extends ImportExportBaseTest
     /** @var SqlCommandBuilder */
     protected $qb;
 
-    /** @var SQLServer2012Platform|AbstractPlatform */
+    /** @var OraclePlatform|AbstractPlatform */
     protected $platform;
```

Comment:
ten platform bude asi k ničemu to sa používalo u Synapse kvůlit quoteSingleIdentifier, ale byl to opruz tak máme ten Quote helper.